receiver/azureblobreceiver/README.md

```diff
@@ -0,0 +1,30 @@
+# Azure Blob Receiver
+
+This receiver reads logs and trace data from [Azure Blob Storage](https://azure.microsoft.com/services/storage/blobs/).
+
+## Configuration
+
+The following settings are required:
+
+- `connection_string` (no default): Azure Blob Storage connection key, which can be found in the Azure Blob Storage resource on the Azure Portal.
+- `event_hub_end_point` (no default): Azure Event Hub endpoint triggering on the `Blob Create` event 
```

Comment:
We usually use just `endpoint` for this. If there could be several endpoints then I would use:

```
event_hub:
  endpoint:
```